invenio_rdm_migrator/streams/records/table_generators/records.py

```diff
@@ -195,8 +207,33 @@ def _generate_rows(self, data, **kwargs):
 
     def _resolve_references(self, data, **kwargs):
         """Resolve references e.g communities slug names."""
-        # resolve parent communities slug
-        parent = data["parent"]
-        communities = parent["json"].get("communities")
-        if communities:
-            self.resolve_communities(communities)
+        if "record" in data:
+            # resolve parent communities slug
+            parent = data["parent"]
+            communities = parent["json"].get("communities")
+            if communities:
+                self.resolve_communities(communities)
+
+    def insert_media_files(self, db_uri=None):
```

Comment:
Media/extra files are stored in legacy in `files_{files,objects,buckets}`, which are already imported "as-is" via CSV dumps. That means we can reuse the same trick for media files (on both drafts and records).

Reply:
@slint can it be generalized into a function and parametrize the insertion table (i.e. `RDMDraftMediaFile` and `RDMDraftFiles`, etc.)

Reply:
Yeah, probably, we could just pass table-names/models and generate the required SQL that way. I was just a bit hesitant to use inheritance in that case, but it could be a mixin or similar.